matisse/src/main/java/com/zhihu/matisse/internal/model/AlbumCollection.java

```diff
@@ -34,13 +34,15 @@
     private LoaderManager mLoaderManager;
     private AlbumCallbacks mCallbacks;
     private int mCurrentSelection;
+    private boolean isLoadFinished;
```

Comment:
Global variable start with ‘m’ is better .  would you please change this ?
 you can checkout codestyle with  command 

**'./gradlew checkstyle '** on mac OS  or  **'gradlew checkstyle'** on Windows  in this project .